app/src/main/java/com/sethpthomas91/httpserver/handlers/ToDoHandler.java

```diff
@@ -0,0 +1,105 @@
+        if (httpRequest.hasHeaders() && hasJsonContentType(httpRequest.getHeaders())) {
+            statusLine.setStatusCode("201");
+            statusLine.setHttpVersion(httpRequest.getRequestLine().getHttpVersion());
+            statusLine.setResponseText("Resource created");
+        } else if (httpRequest.hasHeaders() && hasXFormContentType(httpRequest.getHeaders())){
+            statusLine.setStatusCode("400");
+            statusLine.setHttpVersion(httpRequest.getRequestLine().getHttpVersion());
+            statusLine.setResponseText("Bad request");
+        }else {
+            statusLine.setStatusCode("415");
+            statusLine.setHttpVersion(httpRequest.getRequestLine().getHttpVersion());
+            statusLine.setResponseText("Unsupported media type");
+        }
+        httpResponse.setStatusLine(statusLine);
+        return httpResponse;
+    }
+
+    private boolean hasJsonContentType(RequestHeaders headers) {
+        String contentType = headers.getContentType();
+        return contentType.equals("application/json");
+    }
+
+    private boolean hasXFormContentType(RequestHeaders headers) {
+        String contentType = headers.getContentType();
+        return contentType.equals("application/x-www-form-urlencoded");
+    }
+
+    private HttpResponseWrapper handleBody(HttpRequestInterface httpRequest, HttpResponseWrapper httpResponse) throws IOException {
+        Body body = new Body();
+        if (httpResponse.getStatusLine().getStatusCode() == "201") {
+            body.setBodyText("{\"task\":\"a new task\"}");
+        }
+        httpResponse.setBody(body);
+        return httpResponse;
+    }
+
+    private HttpResponseWrapper handleHeaders(HttpRequestInterface httpRequest, HttpResponseWrapper httpResponse) throws IOException {
+        Header header = new Header(httpRequest, httpResponse);
+        if (httpResponse.getStatusLine().getStatusCode() == "201") {
+            header.setContentType("application/json;charset=utf-8");
+        }
+        httpResponse.setHeaders(header);
+        return httpResponse;
+    }
+
+    private HttpClient createHttpClient() {
+        return HttpClient.newHttpClient();
+    }
+
+    private HttpRequest createHttpRequest(HttpRequestInterface incomingRequest) throws URISyntaxException {
+        HttpRequest request = HttpRequest.newBuilder()
+                .uri(new URI("http://localhost:3000/tasks"))
+                .headers("Content-Type", "application/json;charset=utf-8")
+                .POST(HttpRequest.BodyPublishers.ofString(incomingRequest.getBody()))
+                .build();
+        return request;
+    }
+
+    private HttpResponse<String> sendRequest(HttpClient client, HttpRequest request) throws IOException, InterruptedException {
+        return client.send(request, HttpResponse.BodyHandlers.ofString());
+    }
+
+    private HttpResponse<String> actAsProxy(HttpRequestInterface incomingRequest) throws URISyntaxException, IOException, InterruptedException {
```

Comment:
@scottyplunkett This is the act as proxy method that will be refined for tomorrow's IPM.